infrastructure/environments/README.md

```diff
@@ -31,8 +31,12 @@ Infrastructure which the applications are deployed toInfrastructure which is com
 | k8s\_vm\_size | VM size | `string` | `"Standard_DS2_v2"` | no |
 | key\_vault\_id | Key Vault ID | `string` | n/a | yes |
 | location | Default location for resources | `string` | `"uksouth"` | no |
+| mongodb\_consistency\_max\_interval\_in\_seconds | Represents the amount of staleness that is tolerated (in seconds) - min 5 mins for global replication | `number` | `300` | no |
+| mongodb\_consistency\_policy | Cosmos consistency policy | `string` | `"BoundedStaleness"` | no |
+| mongodb\_databases | List of databases and collections to provision | <pre>list(object({<br>    name = string<br>    collections = list(object({<br>      name = string<br>      default_ttl_seconds = number<br>      indexes = list(object({<br>        keys = set(string)<br>        unique = bool<br>      }))<br>    }))<br>  }))</pre> | n/a | yes |
 | mongodb\_failover\_read\_locations | Locations where read failover replicas are created for MongoDB | `list(string)` | `[]` | no |
-| mongodb\_max\_throughput | Max throughput of the MongoDB database - set in increments of 1,000 between 4,000 and 1,000,000 | `number` | `4000` | no |
+| mongodb\_max\_staleness\_prefix | Represents the number of state requests that are tolerated - min 100,000 for global replication | `number` | `100000` | no |
+| mongodb\_max\_throughput | Max throughput of the MongoDB database - set in increments of 1,000 between 4,000 and 1,000,000 | `number` | `400` | no |
```

Comment:
checking - potential typo here? 

Was previously `4000`, now this has changed to `400` - doc states between 1,000-1,000,000 - is this correct?

```diff
-| `number` | `4000` | no |`
+| `number` | `400` | no |
```

Reply:
Good catch - the documentation is wrong.